data/data-pipeline/data_pipeline/etl/sources/census_decennial/etl.py

```diff
@@ -147,6 +148,65 @@ def __init__(self):
             field_names.CENSUS_DECENNIAL_UNEMPLOYMENT_FIELD_2009
         )
 
+        # Race/Ethnicity fields
+        self.TOTAL_RACE_POPULATION_FIELD = "PCT086001"  # Total
+        self.ASIAN_FIELD = "PCT086002"  # Total!!Asian
+        self.BLACK_OR_AA_FIELD = "PCT086003"  # Total!!Black or African American
+        self.NATIVE_HI_OR_API_FIELD = (
+            "PCT086004"  # Total!!Native Hawaiian and Other Pacific Islander
+        )
+        self.WHITE_FIELD = "PCT086005"  # Total!!White
+        self.HISPANIC_OR_LATINO_FIELD = "PCT086006"  # Total!!Hispanic or Latino
+        self.TWO_OR_MORE_RACES_FIELD = (
+            "P004024"  # Total!!Two or More Ethnic Origins or RaceTotal
+        )
+        self.OTHER_ETHNIC_ORIGIN_FIELD = (
+            "PCT086007"  # Total!!Other Ethnic Origin or Ra
+        )
+
+        self.TOTAL_RACE_POPULATION_VI_FIELD = "P003001"  # Total
+        self.BLACK_VI_FIELD = (
+            "P003003"  # Total!!One race!!Black or African American alone
+        )
+        self.AMERICAN_INDIAN_VI_FIELD = "P003005"  # Total!!One race!!American Indian and Alaska Native alone
```

Comment:
Sorry to be the bearer of bad news, but I think these are off-by-one? 

P003005 looks like `Total!!Asian alone` to me, and `[P003006](https://api.census.gov/data/2010/dec/sf1/variables/P003006.json)` is `Total!!Native Hawaiian and Other Pacific Islander alone`. 

I'm looking at [these](https://api.census.gov/data/2010/dec/sf1/variables.html).

Reply:
I haven't checked the rest of the `P00`s but will do so after you review.

Reply:
So I think we're looking at different documentation.  For the USVI, we're hitting https://api.census.gov/data/2010/dec/vi/variables.html --- that's different than summary file 1.

So when I go to https://api.census.gov/data/2010/dec/vi/variables/P003005.json I see the description "Total!!One race!!American Indian and Alaska Native alone" which does match my variable name

Reply:
Lucas and I talked, and the census helpfully re-uses variable names across products but gives them different meanings. In this case, the vi variables are correct since that's the endpoint we're using and not sf1.